mobile-app/app/styles.json

```diff
@@ -3657,6 +3657,9 @@
 	"mb-5": {
 		"marginBottom": 20
 	},
+	"mb-5.5": {
```

Comment:
let's move this to inline styling since this is a one-off use case

Reply:
Hmm for some reason, I'm not able to add arbitrary value in tailwind:
<img width="324" alt="image" src="https://user-images.githubusercontent.com/17063281/195750419-ac4e5fd0-ab6a-407e-82ef-ae08a77b3860.png">
that's why I just added this here. Any idea how can I add custom value? 🙏

Reply:
not tailwind arbitrary value, but react native inline styling

Reply:
I'm just passing the style as a prop, I just used `mb-5` to get rid of this